### PR TITLE
Add changelog for v1.13.1

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
...
+  * Make sure email recipients can see issue (#13820) (#13827)
+  * Reply button is not removed when deleting a code review comment (#13824)
+  * When reinitialising DBConfig reset the database use flags (#13796) (#13811)
+* ENHANCEMENTS
+  * Add emoji in label to project boards (#13978) (#14021)
+  * Send webhook when tag is removed via Web UI (#14015) (#14019)
+  * Migrations: Use Process Manager to create own Context (#13792) (#13793)
+* API
+  * [API] GetCombinedCommitStatusByRef always return json & swagger doc fixes (#14047)
+  * [API] return original URL of Repositories (#13885) (#13886)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * GetCombinedCommitStatusByRef always return json & swagger doc fixes (#14047)
  * Return original URL of Repositories (#13885) (#13886)
```

**Comment by lunny** [comment]
@jolheiser done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.1](https://github.com/go-gitea/gitea/releases/tag/v1.13.1) - 2020-12-29
+
+* SECURITY
+  * Hide private partisipation in Orgs (#13994) (#14031)
+  * Fix escaping issue in diff (#14153) (#14154)
+* BUGFIXES
+  * Migration: drop too long repo topics (#14152) (#14155)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * Drop long repo topics during migration (#14152) (#14155)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix branch/tag notifications in mirror sync (#13855) (#13862)
+  * Fix crash in short link processor (#13839) (#13841)
+  * Update font stack to bootstrap's latest (#13834) (#13837)
+  * Make sure email recipients can see issue (#13820) (#13827)
+  * Reply button is not removed when deleting a code review comment (#13824)
+  * When reinitialising DBConfig reset the database use flags (#13796) (#13811)
+* ENHANCEMENTS
+  * Add emoji in label to project boards (#13978) (#14021)
+  * Send webhook when tag is removed via Web UI (#14015) (#14019)
+  * Migrations: Use Process Manager to create own Context (#13792) (#13793)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * Use Process Manager to create own Context (#13792) (#13793)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.1](https://github.com/go-gitea/gitea/releases/tag/v1.13.1) - 2020-12-29
+
+* SECURITY
+  * Hide private partisipation in Orgs (#13994) (#14031)
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
  * Hide private participation in Orgs (#13994) (#14031)
```

**Review by jolheiser — COMMENTED** [review]
Other small nits

File: CHANGELOG.md (hunk 4 above)
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Hide private participation in Orgs (#13994) (#14031)
```

**Review by techknowlogick — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,41 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.1](https://github.com/go-gitea/gitea/releases/tag/v1.13.1) - 2020-12-29
+
+* SECURITY
+  * Hide private participation in Orgs (#13994) (#14031)
+  * Fix escaping issue in diff (#14153) (#14154)
+* BUGFIXES
+  * Drop long repo topics during migration (#14152) (#14155)
```
Comment (zeripath) [inline CHANGELOG.md:14]:
```suggestion
  * Fix bug of link query order on markdown render (#14156) (#14171)
  * Drop long repo topics during migration (#14152) (#14155)
```

**Review by zeripath — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Review by 6543 — COMMENTED** [review]
:tada: LGTM

**Pull request merged**